util.sh

```diff
@@ -79,8 +79,13 @@ travis_timeout() {
 # travis_fold (start|end) [name] [message]
 travis_fold() {
   # option -g declares those arrays globally!
-  declare -ag _TRAVIS_FOLD_NAME_STACK  # "stack" array to hold name hierarchy
-  declare -Ag _TRAVIS_FOLD_COUNTERS  # associated array to hold global counters
+
+  if [[ "${TRAVIS_OS_NAME}" == osx ]]; then
```

Comment:
You should check if the `TRAVIS_OS_NAME` is not equal to osx

Reply:
Why should this section be OS-specific at all?
Doesn't osx's bash provide global variables??